servers/republik/graphql/schema-types.js

```diff
@@ -49,6 +49,7 @@ extend type User {
   ): String
 
   birthday: Date
+  yearOfBirth: Int
```

Comment:
why exposed here? no resolver defined right? Will do nothing—always be `undefined`?